docs/pages/versions/unversioned/sdk/location.md

```diff
@@ -21,20 +22,31 @@ import SnackInline from '~/components/plugins/SnackInline';
 
 ### Android permissions
 
-- This module requires the permissions for approximate and exact device location. It also needs the foreground service permission to subscribe to location updates, while the app is in use. The `android.permission.ACCESS_COARSE_LOCATION`, `ACCESS_FINE_LOCATION`, and `FOREGROUND_SERVICE` permissions are automatically added.
-- In order to use background location features, you also must add the `android.permission.ACCESS_BACKGROUND_LOCATION` and [submit your app for review and request access to use the background location permission](https://support.google.com/googleplay/android-developer/answer/9799150?hl=en).
+The `expo-location` module requires permission for approximate (`ACCESS_COARSE_LOCATION`) and precise (`ACCESS_FINE_LOCATION`) device location in the `android.permission` property in **app.json**. It also needs foreground service permission (`FOREGROUND_SERVICE`) to subscribe to location updates while the app is in use. When you install this module in your Expo project, all of the three permissions are added automatically.
```

Comment:
this point is confusing to me - it reads like users will need to add these permissions to their app.json, but they are actually added automatically

Reply:
Good point! If it is confusing, I'll rephrase it to convey what permissions this module automatically adds when using in an Expo project.